contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol

```diff
@@ -699,15 +690,21 @@ contract OracleReportSanityChecker is AccessControlEnumerable {
             emit RequestTimestampMarginSet(_newLimitsList.requestTimestampMargin);
         }
         if (_oldLimitsList.maxPositiveTokenRebase != _newLimitsList.maxPositiveTokenRebase) {
-            _checkLimitValue(_newLimitsList.maxPositiveTokenRebase, type(uint64).max);
+            _checkLimitValue(_newLimitsList.maxPositiveTokenRebase, 1, type(uint64).max);
             emit MaxPositiveTokenRebaseSet(_newLimitsList.maxPositiveTokenRebase);
         }
         _limits = _newLimitsList.pack();
     }
 
     function _checkLimitValue(uint256 _value, uint256 _maxAllowedValue) internal pure {
         if (_value > _maxAllowedValue) {
-            revert IncorrectLimitValue(_value, _maxAllowedValue);
+            revert IncorrectLimitValue(_value, 0, _maxAllowedValue);
+        }
+    }
+
+    function _checkLimitValue(uint256 _value, uint256 _minAllowedValue, uint256 _maxAllowedValue) internal pure {
```

Comment:
Maybe having just a single version for both min and max is better?
Overloads can be dangerous in a long-run :thinking: